test/e2e/utils.js

```diff
@@ -74,9 +99,9 @@ const dockerCommand = (files, ...params) => {
 };
 
 const serviceComposeCommand = (...args) => dockerCommand(DOCKER_COMPOSE_FILE, ...args);
-const testComposeCommand = (fileNames, ...args) => {
-  fileNames = Array.isArray(fileNames) ? fileNames : [fileNames];
-  const filePaths = fileNames
+const testComposeCommand = (filenames, ...args) => {
```

Comment:
Why changing from `fileNames` to `filenames`? 

Comment applies to any occurrence of `filename` in this PR 🙂

Reply:
It's minor and either is technically correct but [nodejs](https://nodejs.org/api/fs.html) and [wikipedia](https://en.wikipedia.org/wiki/Filename) treat it as a single word so I went with that

Reply:
It looks weird to me 😅, but okay, it's readable

Reply:
I'm not too fussed either way; readability and consistency are more important. Let me know if you want me to change it 🙂